contrib/init/systemd/docker.service

```diff
@@ -10,7 +10,6 @@ Type=notify
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
 ExecStart=/usr/bin/docker daemon -H fd://
-MountFlags=slave
```

Comment:
I believe we don't want to change this (At least not for 1.11) in favor of just providing a custom config with the desired behavior.